marklogic-data-hub/src/main/resources/dhf-defaults.properties

```diff
@@ -59,8 +59,8 @@ mlFlowDeveloperUserName=flow-developer
 mlFlowDeveloperPassword=flow-developer
 mlHubLogLevel=default
 
-# Default module permissions which allow flow-operator-role to execute flows
-mlModulePermissions=rest-reader,read,rest-writer,insert,rest-writer,update,rest-extension-user,execute
+# Default module permissions
+mlModulePermissions=data-hub-module-reader,read,data-hub-module-reader,execute,data-hub-module-writer,insert,rest-extension-user,execute
```

Comment:
Just a note that the PR for DHFPROD-3779 defines these in the new AbstractHubConfig class, so they'll need to be updated there as well.

Reply:
Also, data-hub-module-writer,insert should be data-hub-module-writer/update.

Reply:
Changed to data-hub-module-writer/update